packages/qqwing.rb

```diff
@@ -0,0 +1,36 @@
+require 'package'
+
+class Qqwing < Package
+  description 'QQwing is a Sudoku puzzle generator and solver.'
+  homepage 'https://qqwing.com/'
+  @_ver = '1.3.4'
+  version @_ver
+  source_url "https://github.com/stephenostermiller/qqwing/archive/v#{@_ver}.tar.gz"
+  source_sha256 'dc5d61b4470862b190d437f8143d4090639c164d34461c4caa5c081c5c91e6cc'
+
+  def self.patch
+    system 'sed -i s:@build/src-copyright-check.sh::g Makefile'
```

Comment:
Does this pose a licensing violation issue?

Reply:
I am not sure about it (it builds failed with that script)

Reply:
Have you read the copyright text to determine the legal restrictions, if any?